assets/dev/js/admin/hints/media.js

```diff
@@ -0,0 +1,110 @@
+( function() {
+	if ( ! elementorAdminHints || ! elementorAdminHints?.mediaHint ) {
+		return;
+	}
+	if ( ! wp || ! wp.media || ! wp.media.view || ! wp.media.view.Attachment || ! wp.media.view.Attachment.Details ) {
```

Comment:
```suggestion
	if ( ! wp.?media.?view.?Attachment.?Details ) {
```

Reply:
@KingYes 
I think optional chaining is the other way `?.` not `.?`